Modules/IO/NIFTI/src/itkNiftiImageIO.cxx

```diff
@@ -1753,21 +1753,26 @@ Normalize(std::vector<double> & x)
 static bool
 IsAffine(const mat44 & nifti_mat)
 {
-  const vnl_matrix_fixed<float, 4, 4> mat(&(nifti_mat.m[0][0]));
+  vnl_matrix_fixed<double, 4, 4> mat;
+
+  for (int i = 0; i < 4; ++i)
+    for (int j = 0; j < 4; ++j)
+      mat[i][j] = double{ nifti_mat.m[i][j] };
+
   // First make sure the bottom row meets the condition that it is (0, 0, 0, 1)
   {
-    float bottom_row_error = itk::Math::abs(mat[3][3] - 1.0f);
+    double bottom_row_error = std::fabs(mat[3][3] - 1.0);
```

Comment:
Using `std::fabs` is fine to me of course, but was there a specific reason to use it here, instead of `itk::Math::abs`?  Just curious!

Reply:
Because that's what @vfonov dropped in the original thread. Seems ITK uses the  Math::abs universally, so I'll fix that.